packages/platforms/delta/x86-64/x86-64-delta-ag8032/modules/builds/delta-max6620.c

```diff
@@ -0,0 +1,686 @@
+/*
```

Comment:
Can we add this as a kernel driver patch so all platforms can use it?

Reply:
the changes was made in the commit 1124d88b44a979b535a9b885cb044d955e04ae1a